### ISSUE TITLE
Unit Test Fails

### ISSUE THREAD

**Issue by yudiz-mahavir** [christophhagen/LibSignalProtocolSwift#9]
Blow Test cases under `SessionBuilderTests` are failing
```
- testBasicPreKeyV3
- testRepeatBundleMessageV3
- testOptionalOneTimePreKey
```

Most of the time test cases fail while decrypting a message

**Comment by BaranEm** [comment]
Did you find the solution for the fails?

**Comment by BaranEm** [comment]
Found the issue: Christoph made a commit (https://github.com/christophhagen/LibSignalProtocolSwift/commit/c538ddd0408b92e0da64198cc1a7101a7e1eb633) that is not compatible with the current tests. 

Uncomment the changes made by the commit as seen below. The test should succeed now.

`
    /**
     Load the session record for the remote address
     - throws: `SignalError` of type `storageError`
     - returns: The session record
    */
    private func loadSession() throws -> SessionRecord {
    //        guard store.sessionStore.containsSession(for: remoteAddress) else {
    //            throw SignalError(.noSession, "No session for address")
    //        }
        return try store.sessionStore.loadSession(for: remoteAddress)
    }
`

**Comment by christophhagen** [comment]
The tests are fixed with the recent update to 1.2.1.
